### PR TITLE
Add crc to openshift CI

### PR DESCRIPTION
**Pull request by praveenkumar**
Initial attempt to add crc to CI

### CODE REVIEW

File: ci-operator/step-registry/crc/e2e/test/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+approvers:
+- praveenkumar
```
Comment (alvaroaleman) [inline ci-operator/step-registry/crc/e2e/test/OWNERS:2]: Any chance there is more than one owner for this and the remaining owners files?

Comment (praveenkumar) [reply ci-operator/step-registry/crc/e2e/test/OWNERS:2]: @alvaroaleman not right now but will add as we go.

Comment (petr-muller) [reply ci-operator/step-registry/crc/e2e/test/OWNERS:2]: Who will review your changes to this step?

Comment (praveenkumar) [reply ci-operator/step-registry/crc/e2e/test/OWNERS:2]: @petr-muller I need to add on of the team member

**Comment by praveenkumar** [comment]
@petr-muller @alvaroaleman Added team member to the approval and review roles.

**Pull request merged**